src/requests/workspace.jl

```diff
@@ -103,9 +103,8 @@ function request_julia_config(server::LanguageServerInstance)
         ]))
 
     server.format_options = DocumentFormat.FormatOptions(response[1:11]...)
-    server.runlinter = something(response[22], true)
+    server.runlinter = something(response[22], false)
```

Comment:
Well crap. Is it just this setting that vscode sends `null` for `false` on, or is it all of them? I'll go reread the spec, but this smells like a spec-violation on the part of vscode.